src/lib.rs

```diff
@@ -3,12 +3,13 @@ use std::ops;
 use std::ops::Mul;
 use std::ops::{Index, IndexMut};
 
+///An abstraction over a 2D matrix of f64s. the foundation of this library
 #[derive(Debug, Clone, Copy, PartialEq, PartialOrd)]
 pub struct Matrix<const ROWS: usize, const COLS: usize> {
     matrix: [[f64; COLS]; ROWS],
 }
 
-//Matrix with one column
+///A Matrix with only one column
```

Comment:
```suggestion
/// A [`Matrix`] with only one column
```
You can make link references like this.  This works for std:: items like [\`f64\`] above